storage/engine/rocksdb/db.cc

```diff
@@ -1353,15 +1353,9 @@ DBStatus DBOpen(DBEngine **db, DBSlice dir, DBOptions db_opts) {
   // increased read amplification.
   table_options.block_size = db_opts.block_size;
 
-  rocksdb::ColumnFamilyOptions cf_options;
-  cf_options.OptimizeLevelStyleCompaction(db_opts.memtable_budget);
-  // OptimizeLevelStyleCompaction sets no-compression for L0 and
-  // L1. Current benchmarks and tests show no benefit to doing this.
-  for (int i = 0; i < cf_options.compression_per_level.size(); i++) {
-    cf_options.compression_per_level[i] = rocksdb::kSnappyCompression;
-  }
-
-  rocksdb::Options options(rocksdb::DBOptions(), cf_options);
+  rocksdb::Options options(rocksdb::GetOptions(db_opts.memtable_budget));
+  options.IncreaseParallelism(db_opts.num_cpu);
```

Comment:
Mostly the second but generally all of them - I need a comment to explain why `(num_cpu / 2, 1)` was chosen and I think everyone else who's not in the RocksDB internals will appreciate it too. Super useful to know what your reasoning is when you turn random knobs.